new/BLorient8815.xml

```diff
@@ -0,0 +1,275 @@
+                                    <note>The text is divided into five sections. Desinit ex abrupto (VIII, 2).</note>
+                                    <msItem xml:id="ms_i1.2.2.1">
+                                        <title ref="LIT2362Songof#FirstCanticle"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.2">
+                                        <title ref="LIT2362Songof'#SecondCanticle"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.3">
+                                        <title ref="LIT2362Songof#ThirdCanticle"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.4">
+                                        <title ref="LIT2362Songof#FourthCanticle"/>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.2.5">
+                                        <title ref="LIT2362Songof#FifthCanticle" type="incomplete"/>
+                                    </msItem>
+                                </msItem>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">2+35</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>142</height>
+                                        <width>110</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="21"/>
+                            </layoutDesc>    
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Written in good hand.</desc>
+                                <date notBefore="1700" notAfter="1800" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="GuestText">Between the front cover and the flyleaf is a double sheet of paper. On the first part is a Hebrew 
+                                        translation of the Ethiopic Apocryphal Psalm 151.</desc>
```

Comment:
OK. How can I describe this as accMat? Is it correct to include accMat instead of a1 in xml:id?
```
<item xml:id="accMat">
<desc type="GuestText">Between the front cover and the flyleaf is a double sheet of paper. On the first part is a Hebrew translation of the Ethiopic Apocryphal Psalm 151.</desc>
<q xml:lang="he"><title>מזמור שלודיד שכתב בהיותו לבדו ונלחם עם גלית ואיננו ממספר מזמרות ויאמר</title>                               
``` 
I did not mention explicitly, that this text was translated from the Ethiopic text. Do you want me to delete the word 'Ethiopic' in "Ethiopic Apocryphal Psalm"?

Reply:
I added a guidelines passage with an example from a manuscript we have

Reply:
Yes I would delete Ethiopic unless there are other ideas